auth4genai/snippets/how-tos/salesforce/salesforce-prereqs-js.mdx

```diff
@@ -0,0 +1,7 @@
+Before using this example, make sure you:
+- Complete the [User authentication quickstart](/get-started/user-authentication) to create a Next.js app integrated with Auth0.
+- Set the `SALESFORCE_INSTANCE_URL` in your `.env` file.
```

Comment:
this step should probably be after the step "Set up and configure a Salesforce instance [...]".